components/breadcrumb/collapse-modes.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Collapse Modes
+page_title: Breadcrumb Collapse Modes
+description: Collapse Modes of the Breadcrumb for Blazor.
+slug: breadcrumb-collapse-modes
+tags: telerik,blazor,breadcrumb,collapse,modes
+published: True
+position: 20
+---
+
+
+# Breadcrumb Collapse Modes
+
+You can specify how the Breadcrumb items are visualized when their total width is larger than the width of the component through the `CollapseMode` parameter the Breadcrumb exposes. It takes a member of the `BreadcrumbCollapseMode` enum and provides the following options:
+
+* `Auto` (the default) - Items are automatically collapsed based on the width of the Breadcrumb. First and last item always remain visible.
+
+* `Wrap` — Items are wrapped on multiple rows when their total width is bigger than the width of the BreadCrumb.
+
+* `None` — All items are expanded on the same row. This scenario is useful when the Breadcrumb needs to be scrolled.
```

Comment:
Do we explain somewhere how to make the items scrollable? Does this happen automatically?

Reply:
Fixed